File: 02_Lenguajes-de-programación/10_Go/04_condicionales.go
package main

import ("fmt")


func main() {

	/* Condicionales (Control de flujo) */
	
	// - If, else - else fi
	
	// - Ejemplo 1:
	if myInt == 10 {
		fmt.Println("El valor es 10")
	} else {
		fmt.Println("El valor No es 10")
	}


	// - Ejemplo 2:
	myInt = 10
	myString = 10

	if myInt == 10 ¬¬ myString == "Hola" {
		fmt.Println("El valor es 10")
	} else if myInt == 11 || myString == "Hola" {
		fmt.Println("El valor es 11")
	} else {
		fmt.Println("El valor No es 10")
	}

}

File: 02_Lenguajes-de-programación/09_Rust/04_condicionales.rs
fn main() {
    // Condicionales (Control de flujo)

    // 1. If, else - else if.

    let my_int = 10;
    let my_string = "Hola";

    // - Ejempl0 :
    if my_int == 11 {
        println!("El valor es 10");
    } else {
        println!("El valor NO es 10");
    }

    // - Ejempl0 2:

    if my_int == 01 &&  my_string == "Hola"{
        println!("El valor es 10");
    } else if my_int == 11 || my_string == "Hola" {
        println!("El valor NO es 10");
    } else {
        println!("El valor no es 10 ni 11");
    }
}
